EasyPost.Tests.Net/VCR.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using Scotch;
+
+namespace EasyPost.Tests.Net
+{
+    public enum VCRApiKey
+    {
+        Test,
+        Production
+    }
+
+    public static class VCR
+    {
+        public static string GlobalApiKey { get; set; }
+
+        private static string CassetteFolder { get; set; }
+
+        private static bool _HideCredentials { get; set; }
+
+        private static string GetSourceFileDirectory([CallerFilePath] string sourceFilePath = "")
+        {
+            return Path.GetDirectoryName(sourceFilePath);
+        }
+
+        private static string GetCassettePath(string cassetteName)
+        {
+            string cassetteFolder = Path.Combine(GetSourceFileDirectory(), "cassettes");
+            Directory.CreateDirectory(cassetteFolder);
+            if (CassetteFolder != null)
+            {
+                cassetteFolder = Path.Combine(cassetteFolder, CassetteFolder);
+                Directory.CreateDirectory(cassetteFolder);
+            }
+
+            return Path.Combine(cassetteFolder, $"{cassetteName}.json");
+        }
+
+        private static string GetApiKey(VCRApiKey apiKey)
+        {
+            string keyName = "";
+            switch (apiKey)
+            {
+                case VCRApiKey.Test:
+                    keyName = "EASYPOST_TEST_API_KEY";
+                    break;
+                case VCRApiKey.Production:
+                    keyName = "EASYPOST_PROD_API_KEY";
+                    break;
+            }
+
+            string key = Environment.GetEnvironmentVariable(keyName) ?? "couldnotpullapikey"; // if can't pull from environment, will use a fake key. Won't matter on replay.
+            Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
+            Trace.WriteLine("Using API Key: " + key);
```

Comment:
Printing API keys isn't ideal for security reasons. Is it possible we could hide this behind a debug flag or something?

Reply:
I can probably remove it altogether, this was added just to help troubleshoot an issue with the GitHub CI.
That said, I've noticed that the GitHub CI won't print the secret in plain text here, it will print a censored "***"

Reply:
Resolved in 88726861fba93fd348a76b99a851ecbdaf1c49d5